src/service-worker.ts

```diff
@@ -22,7 +22,11 @@ sw.addEventListener('install', (event) => {
         await cache.addAll(ASSETS);
     }
 
-    event.waitUntil(addFilesToCache());
+    // Check if the page URL does not match the exclusion path
+    const shouldCache = !/\/console\/auth\/oauth2\/.*/.test(location.pathname);
```

Comment:
I think we can do `location.pathname.startswith('/console/auth')` and we can also use the `base` variable from the `$service-worker` module import from top

Reply:
Makes sense! Updated it!